scripts/ci-docker-images/releaser/Dockerfile

```diff
@@ -38,7 +38,8 @@ RUN apt-get remove docker docker-engine docker.io
 RUN apt-get update && apt-get install -y docker.io
 
 # Install gems to reduce setup time
-RUN gem install bundler -v '~> 2.0.0'
-COPY checker/Gemfile Gemfile
+RUN gem install bundler -v '~> 2.1.4'
```

Comment:
We should probably make sure the user here is no `sudo` I ran into this issue yesterday.

Reference: https://github.com/timberio/vector/issues/1621

Reply:
I think it might be better solved by changing `scripts/docker-run.sh` rather than the Dockerfile itself. The reason is that we don't know in advance `uid` of the user who is going to run the command in advance.